routes/users.js

```diff
@@ -122,20 +122,18 @@ router.post('/register', function(req, res, next) {
             db.collection(userDB).insert({ 
                     "username": username,
                     "hash": password,
-                    "email": email
+                    "email": email,
+                    "SEARCH": username.toUpperCase()
                 }, function(err){
                     if(err){
                         res.send("Error processing request");
                     }else{
-                        return res.render('register', {
-                            message:"Go to login to sign in",
-                            type: "success"
-                        }); 
+                        res.redirect('/login');
```

Comment:
I think thats a good trade off. Although it would be nice to have something that say the registration was successful. We should look further into this for sure. But for now its good.